docs/reference/hacl/dh/ec/curve25519.md

```diff
@@ -0,0 +1,92 @@
+# Curve25519
+
+## API Reference
+
+`````{tabs}
+````{group-tab} 51
```

Comment:
`51` probably doesn't tell anyone where this can be used.
This is the portable implementation.

Reply:
To be honest... I try to not give too much thought to these things now because the rename will eventually make everything better :-) I have used the 51 description here solely for consistency with the header file. But I agree that there should be some intro tellig what this is :+1: Will do.